eglot-jl.el

```diff
@@ -51,23 +51,8 @@
 An empty string uses the default depot for ‘eglot-jl-julia-command’
 when the JULIA_DEPOT_PATH environment variable is not set."
   :type 'string)
-
-(defcustom eglot-jl-default-environment "~/.julia/environment/v1.2"
```

Comment:
Please bump the major version in the header since removing this customization is a breaking change.

Reply:
OK, makes sense